### ISSUE TITLE
Repository actions: Solution files not found in subdirectories

### ISSUE THREAD

**Issue by Sam13** [coenm/RepoM#209]
With v1.1.18 solution files are no longer found in subdirectories. Just in the repository root directory.

The following snippet in _RepositoryActionsV2.yaml_ behaves differently to v1.0.87 where this worked
```
    solution_files = file.find_files(repository.linux_path, "*.sln");   
```

**Comment by coenm** [comment]
Fixed and created new release 
https://github.com/coenm/RepoM/releases/tag/v1.1.19

**Comment by Sam13** [comment]
@coenm 
Thanks for the quick fix - works again with v1.1.19 👍
